tutorials/matching_stack_run_1.2i.ipynb

```diff
@@ -0,0 +1,514 @@
+    "print(len(obj_matches), len(src_matches))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`matches` is a list of `match` objects, each one containing an observed-true matchd galaxy pair. The code below shows how to work with these, looping over the matches and extracting information to plot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def compare_differences(drp_catalog, matches, observed_filter):\n",
+    "    num_matches = len(matches)    \n",
+    "    # Compare magnitudes for matched objects:\n",
+    "    drp_mag = np.zeros(num_matches, dtype=np.float)\n",
+    "    truth_mag = np.zeros(num_matches, dtype=np.float)\n",
+    "    sep = np.zeros(num_matches, dtype=np.float)\n",
+    "    # Arrays for a quiver plot.\n",
+    "    u = np.zeros(num_matches, dtype=np.float)\n",
+    "    v = np.zeros(num_matches, dtype=np.float)\n",
+    "    for i, match in enumerate(matches):\n",
+    "        drp_mag[i] = match.first['mag_{}'.format(observed_filter)]\n",
+    "        truth_mag[i] = match.second['mag_{}'.format(observed_filter)]\n",
+    "        sep[i] = np.degrees(match.distance)*3600.*1000. # convert deg into milliarcseconds\n",
+    "        u[i] = match.first['coord_ra'] - match.second['coord_ra']\n",
+    "        v[i] = match.first['coord_dec'] - match.second['coord_dec']\n",
+    "    print(\"Number of matches:\", len(matches))\n",
+    "\n",
+    "    # Start a 2x2 panel figure:\n",
+    "    fig = plt.figure(figsize=(8, 8))\n",
+    "    frame_axes = fig.add_subplot(111, frameon=False)\n",
+    "    frame_axes.get_xaxis().set_ticks([])\n",
+    "    frame_axes.get_yaxis().set_ticks([])\n",
+    "\n",
+    "    delta_mag = drp_mag - truth_mag  # Observed - True\n",
+    "    # Upper Left: Histogram of match separations.\n",
+    "    fig.add_subplot(2, 2, 1)\n",
+    "    plt.hist(sep, range=(0, 100), histtype='step', bins=40)\n",
+    "    plt.xlabel('separation (marcsec)')\n",
+    "    plt.ylabel('entries / bin')\n",
+    "\n",
+    "    # Upper Right: Quiver plot of (DRP - galaxy_catalog) positions on the sky.\n",
+    "    fig.add_subplot(2, 2, 2)\n",
+    "    plt.quiver(np.degrees(drp_catalog['coord_ra']),\n",
+    "               np.degrees(drp_catalog['coord_dec']), u, v)\n",
+    "    plt.xlabel('RA (deg)')\n",
+    "    plt.ylabel('Dec (deg)')\n",
+    "\n",
+    "    # Lower left: Difference in magnitudes vs true magnitude (mag_gc).\n",
+    "    fig.add_subplot(2, 2, 3)\n",
+    "    plt.errorbar(truth_mag, delta_mag, fmt='.')\n",
+    "    plt.xlabel('True mag {}_gc'.format(observed_filter))\n",
+    "    plt.ylabel('Mag difference ({0}_truth - {0}_drp)'.format(observed_filter))\n",
+    "\n",
+    "    # Difference in magnitudes vs separation.\n",
+    "    fig.add_subplot(2, 2, 4)\n",
+    "    plt.errorbar(sep, delta_mag, fmt='.')\n",
```

Comment:
If I match the object to the source catalogs, those matches and magnitude deltas both look reasonable and fine.

Reply:
And, in fact, given the difficulties with the truth catalogs for Run 1.2, it may be worth adding an example of matching between the `coadd` and `src` catalogs to show that all of this really does work.